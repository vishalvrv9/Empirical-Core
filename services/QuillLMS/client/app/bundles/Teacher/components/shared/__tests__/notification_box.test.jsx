import React from 'react';
import { shallow } from 'enzyme';

import NotificationBox from '../notification_box';

describe('NotificationBox component', () => {
  it('should render', () => {
    const wrapper = shallow(
      <NotificationBox>
        hi
      </NotificationBox>
    );
    expect(wrapper).toMatchSnapshot();
  });

  it('should render text that is passed as a child', () => {
    const passedText = 'Todays Current Event - 5/18/2017: Major and malicious car crash in Time Square.';
    const wrapper = shallow(
      <NotificationBox>
        {passedText}
      </NotificationBox>
    );
    expect(wrapper.find('p').text()).toBe(passedText);
  });
});
