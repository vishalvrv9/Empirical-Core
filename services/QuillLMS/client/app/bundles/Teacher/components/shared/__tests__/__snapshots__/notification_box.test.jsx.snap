// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotificationBox component should render 1`] = `
<div
  className="notification-box flex-row vertically-centered"
>
  <p>
    <i
      aria-hidden="true"
      className="fa fa-lightbulb-o"
    />
    hi
  </p>
</div>
`;
